src/tf/tf.cpp

```diff
@@ -353,6 +354,23 @@ struct tf_parser
         MIGRAPHX_THROW("MIGraphX does not support mean outside of GlobalAvgPool transformation");
     }
 
+    instruction_ref parse_pack(const std::string&,
+                               const attribute_map& attributes,
+                               std::vector<instruction_ref> args)
+    {
+        // reinterpret as unsqueeze with concat
+        std::vector<instruction_ref> unsqueezed_args;
+        int64_t axis = 0;
+        if(contains(attributes, "axis"))
+            axis = attributes.at("axis").i();
+        std::transform(
+            args.begin(),
+            args.end(),
+            std::back_inserter(unsqueezed_args),
+            [&](instruction_ref arg) { return prog.add_instruction(op::unsqueeze{{axis}}, arg); });
+        return prog.add_instruction(op::concat{static_cast<size_t>(axis)}, unsqueezed_args);
+    }
+
```

Comment:
We haven't considered supporting multiple outputs yet since the protobuf file is slightly different compared to onnx. A node in tf protobuf does not contain a field for outputs, so I will need to see a use case where an operator returns multiple outputs.